Source/WebInspectorUI/UserInterface/Views/ResourceContentView.js

```diff
@@ -280,6 +298,17 @@ WI.ResourceContentView = class ResourceContentView extends WI.ContentView
 
         if (this._createLocalResourceOverrideButtonNavigationItem)
             this._createLocalResourceOverrideButtonNavigationItem.enabled = WI.networkManager.canBeOverridden(this._resource);
+
+        if (this._resource.type === WI.Resource.Type.StyleSheet) {
+            requestAnimationFrame(() => {
+                WI.cssManager.lookupStyleSheetForResource(this._resource, (cssStyleSheet) => {
+                    if (cssStyleSheet.wasMutatedByJS)
+                        this.showBanner(WI.UIString("This style sheet has been modified by CSSOM in JavaScript"));
+                    else
+                        this.hideBanner();
+                });
+            });
```

Comment:
I wasn't sure why `requestAnimationFrame` or equivalent was needed here. Without that, I saw the text editor doesn't show up anymore and it's just blank beneath the banner. I didn't bother trying to find out why.